spec/ruby_llm/providers/ollama/ollama_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'dotenv/load'
+
+RSpec.shared_context 'with configured local Ollama server as sole provider' do
+  before :all do
+    RubyLLM.configure do |config|
+      # NOTE: other tests will need to ensure relevant providers are enabled as needed
+      RubyLLM::Provider.disable_all_providers
```

Comment:
This is better than yesterday (because it now handles all current and future providers), but it does mean that each test will need to ensure relevant providers are enabled, to ensure they work regardless of test run order.